src/manifest.rs

```diff
@@ -12,21 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use eyre::Context;
+use miette::IntoDiagnostic;
+use miette::{miette, Context};
```

Comment:
can be combined imports

Reply:
Weird, probably an artefact of a recent merge.